packages/aws-cdk-lib/aws-rds/test/instance.test.ts

```diff
@@ -194,6 +194,22 @@ describe('instance', () => {
     Template.fromStack(stack).resourceCountIs('Custom::LogRetention', 4);
   });
 
+  test('create DB instance with io2 instance storage', () => {
+    // WHEN
+    new rds.DatabaseInstance(stack, 'Instance', {
+      engine: rds.DatabaseInstanceEngine.MYSQL,
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.MEDIUM),
+      vpc,
+      storageType: rds.StorageType.IO2,
+    });
+
+    // THEN
+    Template.fromStack(stack).hasResourceProperties('AWS::RDS::DBInstance', {
+      Iops: 1000,
+      StorageType: 'io2',
+    });
+  });
+
```

Comment:
There is already a similar test here, so it might be better to write it there.

https://github.com/go-to-k/aws-cdk/blob/0605a8b869aac6bccc9631fcea719895ca397f73/packages/aws-cdk-lib/aws-rds/test/instance.test.ts#L2011

Or maybe we could create tests that are separate from this test (`'gp3 storage type'`) to check at once if the `defaultIops` method outputs iops for each storage type. (Leave the test (`'gp3 storage type'`) to check the `storageThroughput`, and change the test name to another name related to `gp3` and `storageThroughput`.)

For example:
```ts
  test.each([
    [rds.StorageType.STANDARD, 'standard', 2000, undefined],
    [rds.StorageType.GP2, 'gp2', 2000, undefined],
    [rds.StorageType.GP3, 'gp3', 2000, 2000],
    [rds.StorageType.IO1, 'io1', 2000, 2000],
    [rds.StorageType.IO1, 'io1', undefined, 1000],
    [rds.StorageType.IO2, 'io2', 2000, 2000],
    [rds.StorageType.IO2, 'io2', undefined, 1000],
  ])('storage type and IOPS for %s storage type', (inStorageType, outStorageType, inIops, outIops) => {
    new rds.DatabaseInstance(stack, 'Instance', {
      engine: rds.DatabaseInstanceEngine.mysql({ version: rds.MysqlEngineVersion.VER_8_0_30 }),
      vpc,
      storageType: inStorageType,
      iops: inIops,
    });

    Template.fromStack(stack).hasResourceProperties('AWS::RDS::DBInstance', {
      StorageType: outStorageType,
      Iops: outIops ?? Match.absent(),
    });
  });
```

It may not be to your taste, which do you think would be better? Of course, the above code can be modified.

Reply:
@go-to-k Thank you for the excellent suggestion!! I replaced unit test and renamed existing `gp3` test.